### DISCUSSION TITLE
Ability to Auto-Copy Field Content to Other Fields

### DISCUSSION THREAD

**Discussion by gcasasi** [docusealco/docuseal#357] [Ideas]
Hello Docuseal team,

First of all, I want to thank you for creating such an amazing tool. It has been incredibly helpful in managing legal documents efficiently.

I would like to propose a feature that could further enhance the user experience. In many legal texts, the same information (such as a person's name) needs to be entered in multiple places. It would be very convenient if the platform had the option to automatically copy the content from one field to another. This would save time and reduce the possibility of errors in repetitive data entry.

Thank you again for all your hard work on this fantastic tool. I hope you consider this suggestion for future updates.

Best regards,
Gabriel

**Comment by omohokcoj** [comment]
@gcasasi it's possible to add the same field on multiple pages to have the data copied automatically:
https://www.docuseal.co/resources/add-field-on-multiple-pages

**Reply by gcasasi** [reply]
Hello @omohokcoj, I apologize for missing that and appreciate your thorough work on the platform.
Best regards,
Gabriel
